docs/3.0.0-beta.x/getting-started/quick-start.md

```diff
@@ -19,25 +19,26 @@ npm install strapi@beta -g
 ## 2. Create a new project
 
 ```bash
-strapi new cms --quickstart
+strapi new my-project --quickstart
 ```
 
-## 3. Create an admin user
+## 3. Create an Administrator user
 
 Navigate to [**http://localhost:1337/admin**](http://localhost:1337/admin).
 
-- Fill in the form.
+- Complete the form to create an Administrator user.
```

Comment:
Complete the form to create the first Administrator user.

Reply:
this was fixed in the other branch.  I will leave alone here.